config/features.rb

```diff
@@ -4,6 +4,14 @@
   strategy :active_record, class: Hyrax::Feature
   strategy :default
 
+  feature :proxy_deposit,
+          default: true,
+          description: "Depositors may designate proxies to deposit works on their behalf"
+
+  feature :transfer,
```

Comment:
I believe an additional modifier for the feature name could be helpful: `transfer_works`